safe-global/safe-homepage
# fix: Search within project categories

## What it solves

Searches within project categories that are persisted as an array instead of a comma-separated string which didn't work so well with fuzzy search

## How to test

1. Open the Ecosystem page
2. Search for Marketplace
3. Observe the same amount of results as if filtering by the same category
